charts/rancher-monitoring/v0.0.4/charts/exporter-node/values.yaml

```diff
@@ -1,4 +1,5 @@
 enabledRBAC: true
+enabledPSP: true
```

Comment:
Do we need to change Rancher code also? is it ok to always enable PSP?

Reply:
PSP is available from v1.5 and the API remains to be beta from 1.11 to 1.15.
It doesn't matter if the admission controller is enabled or not, these resources can be managed in the app anyway.
So it is ok to always turn this flag on.